dotnet/src/Connectors/Connectors.AzureAIInference/Settings/AzureAIInferencePromptExecutionSettings.cs

```diff
@@ -273,8 +273,8 @@ public static AzureAIInferencePromptExecutionSettings FromExecutionSettings(Prom
     private float? _nucleusSamplingFactor;
     private int? _maxTokens;
     private object? _responseFormat;
-    private IList<string> _stopSequences = [];
-    private IList<ChatCompletionsToolDefinition> _tools = [];
+    private IList<string>? _stopSequences;
+    private IList<ChatCompletionsToolDefinition>? _tools;
```

Comment:
Initialising tools to an empty array causes a Problem in the JSON payload.

@RogerBarreto  Is there a reason why this is being initialised to an empty array?